src/main/java/org/jfrog/bamboo/builder/GradleInitScriptHelper.java

```diff
@@ -267,6 +267,11 @@ private void addClientProperties(ArtifactoryClientConfiguration clientConf, Serv
         clientConf.publisher.setExcludePatterns(buildContext.getExcludePattern());
         clientConf.publisher.setFilterExcludedArtifactsFromBuild(buildContext.isFilterExcludedArtifactsFromBuild());
         if (publishArtifacts) {
+            try {
+                clientConf.publisher.setPublishForkCount(Integer.parseInt(buildContext.getPublishForkCount()));
+            } catch (NumberFormatException ignore) {
```

Comment:
It may be better to fail the build completely if the value is invalid.
Actually, if we switch the UI component into a select-box with numeric values to choose from, this validation will not be necessary.

Reply:
removed